dashboard/app/models/section.rb

```diff
@@ -74,7 +74,7 @@ def add_student(student, move_for_same_teacher: true)
       # new one
       follower.update_attributes!(section: self)
     else
-      follower = Follower.create!(user_id: self.user_id, student_user: student, section: self)
+      follower = Follower.find_or_create_by!(user_id: self.user_id, student_user: student, section: self)
```

Comment:
interesting that there's also a first_or_create, but this seems appropriate here.